src/peripherals/usart.c

```diff
@@ -221,8 +228,16 @@ void usarts_enable(u32 ftdi_baud, u32 uarta_baud, u32 uartb_baud, bool do_precon
   chBSemInit(&uartb_state.claimed, FALSE);
   uartb_state.configured = true;
 
-  all_uarts_enabled = true;
+  if (do_configure_hooks) {
```

Comment:
The radio "preconfigure" hooks are a bit of a hack right now. They must be called before (and only before) any of the normal uart framework is operational. The openlog hooks use the standard claim() / release() mechanism. I think it's clear from the *pre*configure naming.